### ISSUE TITLE
Not mounting shared drive

### ISSUE THREAD

**Issue by pcs333** [plexdrive/plexdrive#353]
I'm trying to mount my shared drive with a newly compiled plexdrive but it is only mounting my main gdrive not my shared drive.
"fuse option --drive-id is not supported, yet" is the message when running.

**Comment by felixbuenemann** [comment]
What is the full plexdrive commandline you are using?

**Comment by pcs333** [comment]
plexdrive -v 3 --chunk-check-threads=16 --chunk-load-threads=16 --chunk-load-ahead=16 --max-chunks=256 mount -o --drive-id=0AAH4-DGT097sUk9PVA /home/pck/mnt/pcs

**Comment by pcs333** [comment]
Also getting an error of ignoring change drive type

**Comment by felixbuenemann** [comment]
Your commandline is wrong, `-o` must be followed by FUSE mount options, like `-o allow_other,read_only`, so omit it, if you want to use the defaults:

```sh
plexdrive mount -v 3 --chunk-check-threads=16 --chunk-load-threads=16 --chunk-load-ahead=15 --max-chunks=256 --drive-id=0AAH4-DGT097sUk9PVA /home/pck/mnt/pcs
```

Also your settings seem a bit excessive, try with `--chunk-check-threads=4 --chunk-load-threads=4 --chunk-load-ahead=3`, that should be enough for streams exceeding 150 MBit/s, for normal streams a load-ahead of 1 or 2 is enough.

**Comment by felixbuenemann** [comment]
> Also getting WARNING: Ignoring change type drive
That's fine, it should probably be logged at INFO level instead of WARNING.

**Comment by pcs333** [comment]
I  will try that now
Many thanks

**Comment by felixbuenemann** [comment]
@pcs333 If it worked, please close the issue.

I've also just released version 5.1.0 which has team drive support: https://github.com/plexdrive/plexdrive/releases/tag/5.1.0

**Comment by kiityman1** [comment]
> @pcs333 If it worked, please close the issue.
> 
> I've also just released version 5.1.0 which has team drive support: https://github.com/plexdrive/plexdrive/releases/tag/5.1.0

Hey, when I enter my secret, the output just freezes. New 5.1.0

**Comment by pcs333** [comment]
> @pcs333 If it worked, please close the issue.
> 
> I've also just released version 5.1.0 which has team drive support: https://github.com/plexdrive/plexdrive/releases/tag/5.1.0

Yeah sorry I forgot to close. Working fine after entering the right commands 👍

**Comment by kiityman1** [comment]
> > @pcs333 If it worked, please close the issue.
> > I've also just released version 5.1.0 which has team drive support: https://github.com/plexdrive/plexdrive/releases/tag/5.1.0
> 
> Hey, when I enter my secret, the output just freezes. New 5.1.0



> > @pcs333 If it worked, please close the issue.
> > I've also just released version 5.1.0 which has team drive support: https://github.com/plexdrive/plexdrive/releases/tag/5.1.0
> 
> Yeah sorry I forgot to close. Working fine after entering the right commands

Hey, mind telling me the command you used? I can't get it working with my shared drive
